validations/quote.test.ts

```diff
@@ -17,33 +20,48 @@ const headers: AxiosRequestHeaders = config.clientApiKey
   : {}
 
 describe('/quote', () => {
-  describe('/out', () => {
```

Comment:
did we just have tests for quote outs previously?

Reply:
yes